account_operating_unit/models/account_move.py

```diff
@@ -223,8 +223,36 @@ def _check_journal_operating_unit(self):
                 move.journal_id.operating_unit_id
                 and move.operating_unit_id
                 and move.operating_unit_id != move.journal_id.operating_unit_id
+            ):
+                # Change journal_id if create move from other model. e.g., sale.order
+                if move._context.get("active_model"):
```

Comment:
```suggestion
                if move._context.get("active_model") and move._context.get("active_model") == 'account.move':
```

Reply:
I think this is safer but in any case your code was working. Thanks.

Reply:
Why do you use `move._context.get("active_model") == 'account.move'`. I think `move._context.get("active_model") != 'account.move'` because If we created invoice from other model, active_model will be that model. This is change journal for not created from account.move directly.

WDYT, Thank for your suggest :)

Reply:
Yes sorry, I meant != . But in any case it is not big deal. I tested functional and it works.